model/license.go

```diff
@@ -164,15 +166,15 @@ func (f *Features) SetDefaults() {
 	}
 
 	if f.GoogleOAuth == nil {
-		f.GoogleOAuth = NewBool(*f.FutureFeatures)
+		f.GoogleOAuth = NewBool(true)
 	}
 
 	if f.Office365OAuth == nil {
-		f.Office365OAuth = NewBool(*f.FutureFeatures)
+		f.Office365OAuth = NewBool(true)
 	}
 
 	if f.OpenId == nil {
```

Comment:
https://github.com/mattermost/mattermost-webapp/pull/11481/files#diff-128483fb9ca10b5b50926c114c9f7b76158c7520591249b993c7d4e8c1c4a051L4945

These two keys are used in OAuth configurations in admin console. Since we're unlocking this feature, these fields are being set to true in my understanding.

Please correct me if these are used anywhere else in the front-end.